### ISSUE TITLE
Error when pending PVCs exist

### ISSUE THREAD

**Issue by timbuchwaldt** [heptio/ark#260]
We see the following error when trying to back up our environment, the first look at it makes us believe it is due to ark trying to back up a pending PVC:

```
time="2017-12-21T12:21:39Z" level=error msg="backup failed" error="error executing custom action: rpc error: code = Unknown desc = unable to get spec.volumeName: key volumeName not found" key=heptio-ark/rh-backup-2 logSource="pkg/controller/backup_controller.go:258"
```

We have one existing PVC that is ready and should be backed up, it contains a volume name: https://dsh.re/d5dd2 - and in contrast the PVC we assume to fail: https://dsh.re/b403f

Deleting the `pending` PVC validates our theory and leads to working backups.

**Comment by ncdc** [comment]
Hi @timbuchwaldt, thanks for reporting. This is actually a duplicate of #254. That will be fixed in v0.7.0, due out in February. If you need it fixed sooner, we could consider backporting to v0.6.x, or you can build and run from the master branch.

**Comment by timbuchwaldt** [comment]
Hey @ncdc, thanks for your swift reply! Great, wilco.
